### PR TITLE
i18n(it): Add new `tutorial/1-setup/3.mdx` translation

### PR DESCRIPTION
**Pull request by nick747**
#### What kind of changes does this PR include?
- Translated content

#### Description

Translated `src/content/docs/en/tutorial/1-setup/3.mdx` to Italian

### CODE REVIEW

**Comment by nick747** [comment]
Can someone check if using 2 headings h1 is ok here?
```astro title="src/pages/index.astro" del={2} ins={3}
<body>
      <h1>Astro</h1>
      <h1>My Astro Site</h1>
</body>
```

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ```astro title="src/pages/index.astro" del={2} ins={3}
+    <body>
+      <h1>Astro</h1>
+      <h1>My Astro Site</h1>
+    </body>
+    ```
+
+3. Controlla l'anteprima sul browser e dovresti vedere il contenuto della tua pagina aggiornato col nuovo testo.
+
+Congratulazioni! Ora sei un sviluppatore Astro.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/3.mdx:58]:
```suggestion
Congratulazioni! Ora sei uno sviluppatore Astro.
```

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+
+<PreCheck>
+  - Fai le tue prime modifiche al tuo nuovo sito
+</PreCheck>
+
+## Modifica la tua home page
+
+1. Nel tuo editor di codice, naviga nella sezione File Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una tab modificabile.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/3.mdx:23]:
```suggestion
1. Nel tuo editor di codice, naviga nella sezione Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una scheda modificabile.
```

**Review by itsmatteomanf — CHANGES_REQUESTED** [review]
Direi che ci siamo praticamente, due piccolissime cose...

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+
+<PreCheck>
+  - Fai le tue prime modifiche al tuo nuovo sito
+</PreCheck>
+
+## Modifica la tua home page
+
+1. Nel tuo editor di codice, naviga nella sezione Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una scheda modificabile.
```
Comment (ematipico) [inline src/content/docs/it/tutorial/1-setup/3.mdx:23]: The proposition "a" doesn't sound right to me. I would suggest reassembling the phrase, for example, "apri il file `src/pages/index.astro` dalla sezione Explorer, e cliccaci sopra per aprire il suo contentuo in una scheda modificabile"

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      </head>
+      <body>
+        <h1>Astro</h1>
+      </body>
+    </html>
+    ```
+
+2. Modifica il contenuto del `<body>` della tua pagina.
+
+    Inserisci nell'editor per modificare il testo del titolo sulla tua pagina e salva le modifiche.
```
Comment (ematipico) [inline src/content/docs/it/tutorial/1-setup/3.mdx:47]: I think there's something wrong with this phrase, it doesn't feel right. We should rephrase. What's the original phrase? I might be able to suggest an alternative.

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+    ```astro title="src/pages/index.astro" del={2} ins={3}
+    <body>
+      <h1>Astro</h1>
+      <h1>My Astro Site</h1>
+    </body>
+    ```
+
+3. Controlla l'anteprima sul browser e dovresti vedere il contenuto della tua pagina aggiornato col nuovo testo.
```
Comment (ematipico) [inline src/content/docs/it/tutorial/1-setup/3.mdx:56]:
```suggestion
3. Controlla l'anteprima nel browser, e dovresti vedere il contenuto della tua pagina aggiornato col nuovo testo.
```

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+Congratulazioni! Ora sei uno sviluppatore Astro.
+
+Il resto di questa unità ti preparerà per il successo con il controllo di versione e un sito web pubblicato che puoi mostrare.
+
+
+
```
Comment (ematipico) [inline src/content/docs/it/tutorial/1-setup/3.mdx:63]:
```suggestion

```

**Review by ematipico — CHANGES_REQUESTED** [review]

File: src/content/docs/it/tutorial/1-setup/3.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+      <h1>Astro</h1>
+      <h1>My Astro Site</h1>
+    </body>
+    ```
+
+3. Controlla l'anteprima sul browser e dovresti vedere il contenuto della tua pagina aggiornato col nuovo testo.
+
+Congratulazioni! Ora sei uno sviluppatore Astro.
+
+Il resto di questa unità ti preparerà per il successo con il controllo di versione e un sito web pubblicato che puoi mostrare.
```
Comment (csarnataro) [inline src/content/docs/it/tutorial/1-setup/3.mdx:60]: This translation seems too "literal" to me. The original sentence is:
```
The rest of this unit will set you up for success with 
version control and a published website you can show off.
```
I would rephrase it to something more fluent, like:
```
Nel seguito di questa unità imparerai a utilizzare con successo 
i sistemi di controllo di versione e a pubblicare un sito web 
da mostrare al mondo.
```
Of course this is just a suggestion, feel free to adapt it as per your preferences.

File: src/content/docs/it/tutorial/1-setup/3.mdx (hunk 5 above)
Comment (csarnataro) [reply src/content/docs/it/tutorial/1-setup/3.mdx:56]: The comma followed by `e` in "nel browser, e dovresti" doesn't sound good to me. I would simply remove the `e`, e.g. "Controlla l'anteprima nel browser, dovresti vedere..."

File: src/content/docs/it/tutorial/1-setup/3.mdx (hunk 4 above)
Comment (davidochoadev) [reply src/content/docs/it/tutorial/1-setup/3.mdx:47]: @ematipico This is the original phrase:"Type in the editor to change the heading text on your page and save the change. "

I believe the most relevant literary translation can be this: "Digita nell'editor per cambiare il testo dell'intestazione sulla tua pagina e salva la modifica.", but I don't really like it that much.

Comment (csarnataro) [reply src/content/docs/it/tutorial/1-setup/3.mdx:47]: Or, rephrasing a little bit the original sentence: "Modifica il file nell'editor, aggiorna l'intestazione della pagina e infine salva il file".

Comment (ematipico) [reply src/content/docs/it/tutorial/1-setup/3.mdx:47]: +1 for @csarnataro's suggestion!

**Review by csarnataro — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌
